migrations/create.sql

```diff
@@ -1,8 +1,9 @@
+-- note: this schema is only used for postgres_logger
 CREATE SCHEMA lite_rpc;
 
 CREATE TABLE lite_rpc.Txs (
   id SERIAL NOT NULL PRIMARY KEY,
-  signature CHAR(88) NOT NULL,
+  signature TEXT NOT NULL,
```

Comment:
why text, char 88 was correct right.

Reply:
CHAR is not recommended at all AFAIK.
plus it pads the value if it's smaller than 88.

Reply:
you could use VARCHAR which gives you the bonus of check for length limit

Reply:
from docs (https://www.postgresql.org/docs/current/datatype-character.html):
>  In most situations text or character varying should be used instead.